lib/Core/Executor.cpp

```diff
@@ -1483,6 +1483,9 @@ void Executor::transferToBasicBlock(BasicBlock *dst, BasicBlock *src,
     PHINode *first = static_cast<PHINode*>(state.pc->inst);
     state.incomingBBIndex = first->getBasicBlockIndex(src);
   }
+  if (INTERPOLATION_ENABLED)
+	  //blockCount increased to count all visited Basic Blocks
```

Comment:
@rasoolmaghareh The indentation here should be 4 spaces less. You may want to try to use `git clang-format .` to automatically standardize the format of the updates. To configure `git` to be able to use `clang-format` command, you can put the [git-clang-format script](https://llvm.org/svn/llvm-project/cfe/trunk/tools/clang-format/git-clang-format) somewhere in your PATH. To format your updates, first add all the updated files, then say `git clang-format .` You may need to execute this command several times until there is no more change.